ignite/handlers/state_param_scheduler.py

```diff
@@ -43,16 +47,35 @@ def attach(
 
         """
         if hasattr(engine.state, self.param_name):
-            raise ValueError(
-                f"Attribute: '{self.param_name}' is already defined in the Engine.state."
-                f"This may be a conflict between multiple StateParameterScheduler handlers."
-                f"Please choose another name."
-            )
+            if not self.create_new:
```

Comment:
I do not quite understand this if/else clauses, especially "else" one.
I was thinking about the following logic:

1) `param_name` NOT in `engine.state`  AND create_new is True => simply create new attribute
2) `param_name` NOT in `engine.state`  AND create_new is False => warn that we will create new attribute, but to remove this warning, create_new should be True
3) `param_name` in `engine.state`  AND create_new is True => raise error as we can not create new attribute as it already exists in the state.
4) `param_name` in `engine.state`  AND create_new is False => silently override existing attribute

What do you think ?

Reply:
Oh yes , I've misunderstood the case 3, thought we were handling the creation  of state parameter in that case ... So yes Indeed, it's better and simpler ! Thanks